WordPress/Classes/Services/MediaCoordinator.swift

```diff
@@ -606,6 +602,22 @@ extension MediaCoordinator: MediaProgressCoordinatorDelegate {
     }
 }
 
+extension MediaCoordinator: Uploader {
+    func resume() {
+        let service = MediaService(managedObjectContext: mainContext)
+
+        service.getFailedMedia { [weak self] mediaArray in
+            guard let self = self else {
+                return
+            }
+
+            for media in mediaArray {
```

Comment:
A suggestion for a 1 liner: 
`mediaArray.forEach {self.retryMedia($0)}`

Reply:
Changed.